docs-release-notes.md

```diff
@@ -1,3 +1,12 @@
+# June 8, 2023
+
+## GraphQL
+
+- Atlas GraphQL API: Note in Limitation section about when you can safely ignore schema title warnings
```

Comment:
When I list page names for docs page that have had changes, I try to match what's in the table of contents to make it easy for folks to navigate to a docs page that has had updates. In this case, there isn't a table of contents entry that says "Atlas GraphQL API", so I'd call this one "Landing Page" just so it's clear to the reader which page we're talking about.
```suggestion
- Landing Page: Note in Limitation section about when you can safely ignore schema title warnings
```